primeqa/components/base.py

```diff
@@ -27,7 +27,7 @@ def eval(self, *args, **kwargs):
         pass
 
     @abstractmethod
-    def predict(self, *args, **kwargs):
+    def rerank(self, *args, **kwargs):
```

Comment:
The `predict` method in Component seems to have been removed.  Was this intended ?
I think we want to have  `predict` as it was in Component and `rerank `added to the Reranker only

Reply:
I changed only in BaseReranker but @franzmpub changed here. If it's not necessary we have to revert back.